resources/validations/src/ssp.sch

```diff
@@ -802,8 +840,9 @@ A FedRAMP SSP must incorporate a procedure document for each of the 17 NIST SP 8
                         validation-details link must be in accord with its sibling validation-reference.</sch:assert>
         </sch:rule>
     </sch:pattern>
-    <sch:pattern see="https://github.com/18F/fedramp-automation/blob/master/documents/Guide_to_OSCAL-based_FedRAMP_System_Security_Plans_(SSP).pdf page 12">
-
+    <sch:pattern id="fips-199"
+                 see="https://github.com/18F/fedramp-automation/blob/master/documents/Guide_to_OSCAL-based_FedRAMP_System_Security_Plans_(SSP).pdf page 12">
```

Comment:
Looks like we could link directly with `#page=12` appended to the URL - but Github doesn't honor that in it's PDF browser. The alternative could be URLs like this:

https://raw.githubusercontent.com/18F/fedramp-automation/master/documents/Guide_to_OSCAL-based_FedRAMP_System_Security_Plans_(SSP).pdf#page=12

... but Github is setting the content disposition to attachment, triggering a download.

I'm thinking we could add these documents to the static site build, so they may be directly linked to at the Federalist-hosted URLs. Thoughts @GaryGapinski @ohsh6o ? Generally, all our documentation and build artifacts could also be available, if we refashioned the Federalist site as a "fedramp-automation" site rather than just the validations.

Reply:
The Federalist build would include whatever PDF happens to be on the branch it's built from - but it would get tricky to deal with changing root URLs. Might just need to always link to the master branch of GSA, so something like `https://validations.fedramp.gov/documents/Guide_to_OSCAL-based_FedRAMP_System_Security_Plans_(SSP).pdf#page=12`